### DISCUSSION TITLE
Why can't we use Fedora

### DISCUSSION THREAD

**Discussion by airtonix** [GameServerManagers/LinuxGSM#4193] [Q&A]
Tried searching but there's just epic walls of text and no simple answer.

what tools are you guys using that can't be used on Fedora? 

I mean really?? what is this tool doing? wget, cp??? how exotic can the required tools be?

**Comment by dgibbs64** [comment]
You can use fedora. The only thing you might not get is the dependency checking part.

**Comment by dgibbs64** [comment]
My efforts are focused on ensuring our most popular distros have dependency checking available. This includes Ubuntu, Debian and RHEL derivatives like CentOS, Alma and Rocky. Fedora is not commonly used as a server and updates every 6 months which means I have to add dependency support to new version every 6 months on a distro that is not a popular choice for LinuxGSM. 

To put this in to context I currently have 2 Fedora 37 servers checking in. https://linuxgsm.com/data/usage/
